hibernate-reactive-core/src/main/java/org/hibernate/reactive/context/impl/VertxContext.java

```diff
@@ -36,10 +36,10 @@ public void injectServices(ServiceRegistryImplementor serviceRegistry) {
 
 	@Override
 	public <T> void put(Key<T> key, T instance) {
-		final io.vertx.core.Context context = Vertx.currentContext();
+		final ContextInternal context = ContextInternal.current();
 		if ( context != null ) {
 			if ( trace ) LOG.tracef( "Putting key,value in context: [%1$s, %2$s]", key, instance );
-			context.putLocal( key, instance );
+			context.localContextData().put( key, instance );
```

Comment:
that should be removed in favour of Vert.x 5 context local implementation.

Reply:
I didn't know this was being deprecated in Vert.x 5 (well it happened on the same day... https://github.com/eclipse-vertx/vert.x/commit/90fa901d2ef0fea8c5af05ed95e39a2512176810)

@DavideD I'll send a follow-up PR with the necessary changes

Reply:
Thanks